package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/exchange-wrappers",
-  "version": "0.0.9",
+  "version": "0.0.10",
```

Comment:
this is basically a breaking change so publish as 0.1.0

Reply:
yeah good point